specifyweb/specify/management/commands/base_specify_migration.py

```
@@ -10,43 +11,70 @@
+        conn = connections[alias]
+        with transaction.atomic(using=alias):
+            with conn.cursor() as cursor:
+                # Check django table
+                try:
+                    cursor.execute("""
+                        SELECT 1
+                        FROM django_migrations
+                        LIMIT 1;
+                    """)
+                    exists = True
+                except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)